File: count.php
<?php

echo "PHP language\n";
$start_time = microtime(true);
$i = 0;
while ($i < 1000000000) {
    $i++;
}
$end_time = microtime(true);
echo "time: " . ($end_time - $start_time) . " seconds\n";

File: count.kt
fun main() {
    println("Kotlin language")

    val startTime = System.currentTimeMillis()
    var i = 0
    while (i < 1000000000) {
        i++
    }
    val endTime = System.currentTimeMillis()
    println("time: ${(endTime - startTime) / 1000.0} seconds")
}
